scripts/README.md

```
@@ -0,0 +1,101 @@
+```yaml
+  lint_gyro:
+    name: Lint gyro configuration
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v5
+      - run: ./scripts/gyro_report.py --check
+```
+
+### Options
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)